### PR TITLE
Change format of the company address to be displayed in CompanyPage.

### PR DESCRIPTION
**Pull request by laurandarius**

### CODE REVIEW

**Comment by laurandarius** [comment]
[Fix format of the company address to be displayed in CompanyPage.](https://trello.com/c/5BPr1LJK/18-fix-format-of-the-company-address-to-be-displayed-in-companypage)

File: app/containers/ViewCompany/actions.js [hunk 1]
```diff
@@ -9,10 +9,17 @@ export const getCompany = id => ({
   id,
 });
 
-export const getCompanySuccess = company => ({
-  type: GET_COMPANY_SUCCESS,
-  company,
-});
+export const getCompanySuccess = fetchedCompany => {
+  const { address, ...company } = fetchedCompany;
+  company.address = `${address.streetName} ${address.streetNumber}, ap. ${
```
Comment (lgnstn) [inline app/containers/ViewCompany/actions.js]: Don't do this here. Let the component that displays the address to decide how it looks like.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn updated.

File: app/components/Company/index.js [hunk 2]
```diff
@@ -38,21 +38,29 @@ const Company = props => {
     );
   };
 
-  const renderCompany = () =>
-    company ? (
+  const renderCompany = () => {
```
Comment (lgnstn) [inline app/components/Company/index.js]: Please make this component a class. Then this method can be the `render` method. `renderSlideShow` will be a method in this class that you call from the `render` method.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn updated.

**Review by lgnstn — APPROVED** [review]

**Pull request merged**